docs/website/src/pages/examples/botonic-example.js

```diff
@@ -0,0 +1,63 @@
+/* eslint-disable import/no-unresolved */
+// eslint-disable-next-line node/no-missing-import
+import Head from '@docusaurus/Head'
+import React from 'react'
+
+import Analytics from '../../components/analytics'
+
+const BotonicExample = ({ title, rootId, runtimeOptions, src }) => {
+  if (typeof window !== 'undefined') {
+    const removejscssfile = (filename, filetype) => {
+      const targetelement =
+        filetype == 'js' ? 'script' : filetype == 'css' ? 'link' : 'none' //determine element type to create nodelist from
+      const targetattr =
+        filetype == 'js' ? 'src' : filetype == 'css' ? 'href' : 'none' //determine corresponding attribute to test for
+      const allsuspects = document.getElementsByTagName(targetelement)
+      for (let i = allsuspects.length; i >= 0; i--) {
+        //search backwards within nodelist for matching elements to remove
+        if (
+          allsuspects[i] &&
+          allsuspects[i].getAttribute(targetattr) != null &&
+          allsuspects[i].getAttribute(targetattr).indexOf(filename) != -1
+        )
+          allsuspects[i].parentNode.removeChild(allsuspects[i]) //remove element by calling parentNode.removeChild()
+      }
+    }
+    removejscssfile('styles.css', 'css') // Dev
+    removejscssfile('styles.9c057c15.css', 'css') // Prod
+    window.onload = () => {
+      setTimeout(() => {
+        // eslint-disable-next-line no-undef
+        Botonic &&
+          // eslint-disable-next-line no-undef
+          Botonic.render(document.getElementById(rootId), runtimeOptions)
+      }, 500)
```

Comment:
Could this timeout fail in any case? Why half a second? I guess we're simulating a `$(document).ready()` here, right? Maybe `0` would be enough?

Reply:
good question

Reply:
yes, at first it was defined to 0, but it seems that sometimes 0 is not enough (Botonic script is not ready to use), so I have updated it to half a second. I have tested this in the production build and seems to work as expected now.